google/cloud/opentelemetry/integration_tests/monitoring_exporter_integration_test.cc

```diff
@@ -64,38 +60,29 @@ void InstallExporter(
   auto resource = opentelemetry::sdk::resource::Resource::Create(
       {{sc::kServiceNamespace, "gl-cpp"},
        // Host ID is meant to uniquely identify a VM. We don't care, though.
-       {sc::kHostId, "monitoring_exporter_integration_test"}},
-      kSchemaVersion);
+       {sc::kHostId, "monitoring_exporter_integration_test"}});
```

Comment:
FWIW, `generic_task` would require less hacking I think.

Reply:
I agree that `generic_task` would be more straightforward. Thanks.